src/integration/java/io/pinecone/integration/controlPlane/pod/CollectionErrorTest.java

```diff
@@ -5,6 +5,7 @@
 import io.pinecone.exceptions.PineconeException;
 import io.pinecone.helpers.RandomStringBuilder;
 import io.pinecone.proto.VectorServiceGrpc;
+import org.junit.Ignore;
```

Comment:
You need to use the annotation on the class / specific functions. The tests are still running in CI.

I used `org.junit.jupiter.api.Disabled` in one of my tests in this file, ln 129 in this file. You can also pass a string to it with an explanation for why it's disabled/ignored.

You're disabling these tests because of flakiness, would you like me to look at cleaning them up and trying to make them a bit more consistent?

Reply:
I did enable all of them back but yes, let's talk offline on how we can clean it up. I have a few tickets in the roadmap that will help us clean this up.